src/test/java/jp/co/future/uroborosql/expr/spel/SpelExpressionParserTest.java

```diff
@@ -0,0 +1,152 @@
+package jp.co.future.uroborosql.expr.spel;
+
+import static org.hamcrest.CoreMatchers.*;
+import static org.hamcrest.MatcherAssert.*;
+
+import java.sql.DriverManager;
+import java.time.Duration;
+import java.time.Instant;
+import java.time.LocalTime;
+import java.time.format.DateTimeFormatter;
+import java.util.Arrays;
+import java.util.HashSet;
+import java.util.Set;
+
+import org.hamcrest.Matchers;
+import org.junit.Before;
+import org.junit.Test;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+
+import jp.co.future.uroborosql.UroboroSQL;
+import jp.co.future.uroborosql.config.SqlConfig;
+import jp.co.future.uroborosql.context.SqlContext;
+import jp.co.future.uroborosql.context.SqlContextFactoryImpl;
+import jp.co.future.uroborosql.context.test.TestEnum1;
+import jp.co.future.uroborosql.dialect.Dialect;
+import jp.co.future.uroborosql.expr.Expression;
+import jp.co.future.uroborosql.expr.ExpressionParser;
+import jp.co.future.uroborosql.parser.ContextTransformer;
+import jp.co.future.uroborosql.parser.SqlParser;
+import jp.co.future.uroborosql.parser.SqlParserImpl;
+
+public class SpelExpressionParserTest {
```

Comment:
@HidekiSugimoto189 

Since it is almost the same as `OgnlExpressionParserTest`, I think whether to create a parent class.

Reply:
That's true.
Create a parent class.